src/bootloader/bootloaderlite.cc

```diff
@@ -43,6 +43,8 @@ void BootloaderLite::installNotify(const Uptane::Target& target) const {
       }
       const std::string& target_version{std::get<0>(target_version_val)};
 
+      env_cmd_vars_ =  boost::str(boost::format("%s=%s") % OstreeTargetPathVar %
```

Comment:
I think the `case RollbackMode::kFioEFI` is missing at line #37.
Also, it can be something like:

```
case RollbackMode::kFioEFI:
  env_cmd_vars_ =  boost::str(boost::format("%s=%s") % OstreeTargetPathVar...
case RollbackMode::kUbootMasked:
case RollbackMode::kFioVB: {
  ...
}
```

Reply:
correct, fixed